data/human/news.txt

```diff
@@ -51,6 +69,27 @@ news "human merchant in human space"
 			"My first ship was a Sparrow. How I survived my first week in space, I'm still not sure!"
 			"My first ship was a Shuttle. I kind of miss that glorified tour bus."
 			"I love doing Syndicate jobs. Easy money for not much work. I guess they're all so rich that they can afford to pay for the best."
+			"Why does the Deep restrict the sale of City-Ships? They don't seem that much better than other warships or freighters."
+			"I just don't get why anyone would pick the pirate life. Sure, no rules, but any government that doesn't force you to have 10 licenses just to look at a heavy warship is free enough to me."
+			"I've heard rumors of captains seeing the distinct flash of whatever Quarg ships use to jump near the core. Wonder what the Quarg are doing up there."
+			"Being a pirate must be a lose-lose scenario. You do badly and you end up enslaved to a stronger pirate. You do well and you get enough attention to get targeted by the Navy or Militia."
+			"I think that people are too harsh towards pirates. Not all of them are heartless."
+			"Be careful when taking suspicious jobs. You might end up at the wrong end of a gun."
+			"Always check the spaceport for jobs. Some of them pay quite well."
+			"Falcons are the best! They're faster and sleeker than any other civilian warship!"
+			"Leviathans are the best! They're tankier and can hold more firepower than any other civilian warship!"
+			"Why would anyone use a Falcon? They have specifications worse than the Leviathan in every way, and they're more expensive to boot."
+			"How could anyone use a Leviathan? They're dreadfully slow and an eyesore to look at."
+			"People may tell you otherwise, but I think that Vanguards are the best. They have such powerful frontal weaponry."
+			"Warships may have both speed and weaponry on their side, but if you're looking for something to last, use a Behemoth."
+			"I don't get how people could get worked up over what ship to use. As long as I'm in my trusty Protector, I'm happy."
+			"Ugh. Why are fighter pilots so gun-ho about charging right into the guns of their enemies? At the very least, don't make we waste hundreds of thousands of credits replacing my ship when you kill yourself."
+			"Don't you think it's strange that the Navy lets you buy their fighters but not any of their other ships?"
```

Comment:
Well, the Dropship is a "fighter" that is now license locked. Not really a true fighter, though.